lux/executor/PandasExecutor.py

```diff
@@ -317,14 +318,20 @@ def compute_dataset_metadata(self, ldf:LuxDataFrame):
         self.compute_data_model(ldf)
 
     def compute_data_type(self, ldf:LuxDataFrame):
+        from pandas.api.types import is_datetime64_any_dtype as is_datetime
+        temporal_var_list = ["month", "year","day","date","time"]
+
         for attr in list(ldf.columns):
-            temporal_var_list = ["month", "year","day","date","time"]
-            if (isinstance(attr,pd._libs.tslibs.timestamps.Timestamp)): 
+            if is_datetime(ldf[attr]):
+                ldf.data_type_lookup[attr] = "temporal"
+            elif (isinstance(attr,pd._libs.tslibs.timestamps.Timestamp)): 
                 # If timestamp, make the dictionary keys the _repr_ (e.g., TimeStamp('2020-04-05 00.000')--> '2020-04-05')
                 ldf.data_type_lookup[attr] = "temporal"
             # elif any(var in str(attr).lower() for var in temporal_var_list):
             elif str(attr).lower() in temporal_var_list:
```

Comment:
should we get rid of this check now that we have the more robust datetime checker?

Reply:
Line 333 was the more robust datetime checker, but that is for detecting string columns that should be datetime. This line was for detecting columns that the users have already casted to datetime, so this line would still be necessary.

Reply:
Ok this makes sense, I think I was referring to the string check on `temporal_var_list`, see above comment.